pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -342,6 +345,137 @@ func setupKernelTunables(option KernelTunableBehavior) error {
 	return utilerrors.NewAggregate(errList)
 }
 
+func setQOSCgroupMemoryLimit(cgroupManager CgroupManager, qosClass v1.PodQOSClass, limit int64) error {
+	glog.V(2).Infof("[Container Manager] Setting %s QoS memory limit to %d bytes", qosClass, limit)
+	// Attempt to set specified limit
+	cgroupConfig := CgroupConfig{
+		Name: cgroupManager.CgroupName(string(qosClass)),
+		ResourceParameters: &ResourceConfig{
+			Memory: &limit,
+		},
+	}
+	err := cgroupManager.Update(&cgroupConfig)
```

Comment:
i'm thinking the absence of the error message is implicitly logging success.  i am already logging quite a bit every 10s @ V(2)